toolbox/load_agol_layer/src/settings.js

```diff
@@ -0,0 +1,81 @@
+const DEFAULT_ESRI_QUERY_PARAMS = {
```

Comment:
thank you for breaking out these params and documenting them! really help with understanding the rationale behind the values in the request. 🚀 🙌 there are some TILs in here for me.